### ISSUE TITLE
ModelicaSystem.buildModel() does not show build warnings

### ISSUE THREAD

**Issue by bilderbuchi** [OpenModelica/OMPython#145]
If you use the ModelicaSystem API, when `buildModel()` is run, content (e.g. build warnings) returned by GetErrorString gets swallowed if no compilation error occurs.
Probably [this block](https://github.com/OpenModelica/OMPython/blob/7981c3deda64a94b0081a73eb0d8e6399d5187f5/OMPython/__init__.py#L924-L926) should get an elif clause that prints the error string if it's not empty.

**Comment by arun3688** [comment]
@bilderbuchi what kind of warnings is currently not displayed, can you give an example, because the buildModel() API outputs two results

### successful build
```
>>> omc.sendExpression("buildModel(Test)")
('c:/juliatest/Test.jl/extendsbug/Test', 'Test_init.xml')
```
### non successfull build
```
>>> omc.sendExpression("buildModel(Test1)")
('', '')
```
It should always return empty string in case of non successful builds and then we call "getErrorString()" and display the error message, Do you have some example where we miss such warnings?

**Comment by bilderbuchi** [comment]
Sure! For example, the model
```
model test_OMPython_145
  Real x(each start = 1.0); 
equation
  x = 1;
end test_OMPython_145;
```
on simulation in OMEdit outputs 
> [1] 19:24:23 Translation Warning
> [test_OMPython_145: 2:15-2:26]: 'each' used when modifying non-array element x.

When using ModelicaSystem, I don't get any output during the build stage (when the model is instantiated). What's worse, the error string is queried, but then discarded, so I can't even get at the warning by manually querying for the error:
```from OMPython import ModelicaSystem
mod=ModelicaSystem("test_OMPython_145.mo","test_OMPython_145")
print('after building it')
print(mod.requestApi('getErrorString'))
print('done')
```
output:
```
after building it

done
```

**Comment by sjoelund** [comment]
The errors should not be printed; then you won't be able to process them. Better would be to not discard them or return them.

**Comment by bilderbuchi** [comment]
I agree. However OMPython is full of `print` for error reporting, so I thought it preferable to align with the present style, instead of doing it "better", but differently.

**Comment by sjoelund** [comment]
> I agree. However OMPython is already full of `print`s for error reporting, so I thought it preferable to align with the present style, instead of doing it "better", but differently.
> 
> I think improving the situation towards more python-idiomatic code would be a separate issue.

Well... I never use those API functions (only sendExpression) because of these problems. The API is not really possible to use :(

**Comment by bilderbuchi** [comment]
I think the ModelicaSystem API is useful (even if it could use some improvements) because it offers a ~pythonic interface that hides (needless) scripting details from the user.

One thing I could not find out how to do using sendExpression alone is to (re-)simulate, optionally with overrides, without incurring a build stage that seems to always happen when using `simulate()` -- indeed that seems corroborated by the docstring "simulates a modelica model by generating c code, build it and run the simulation executable".

**Comment by sjoelund** [comment]
For that, I use `buildModel` followed by the `subprocess` module to start the simulation since there is no resimulate API

**Comment by bilderbuchi** [comment]
> there is no resimulate API

good to know, tack saa mycket!
